modules/util/source-formatter/src/test/resources/com/liferay/source/formatter/dependencies/expected/upgrade/UpgradeCatchAllCheck.testjava

```diff
@@ -89,6 +89,12 @@ public class UpgradeCatchAllCheck {
 		indexer.getSearchClassNames();
 	}
 
+	public void method(long commerceOrderId) {
+		_commerceOrderItemLocalService.deleteCommerceOrderItems(0, commerceOrderId);
+		CommerceOrderItemLocalServiceUtil.deleteCommerceOrderItems(0, commerceOrderId);
+		CommerceOrderItemLocalServiceUtil.deleteCommerceOrderItems(123);
+	}
+
```

Comment:
@TamyrisTorres Since this parameter is also in the other method you can add these tests to the other method. It will be only one method with all three types of tests.